### PR TITLE
Integration for Gumroad

### PR DESCRIPTION
**Pull request by Tammilore**

### CODE REVIEW

File: docs-v2/integrations/all/gumroad.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Gumroad
+sidebarTitle: Gumroad
+---
+
+API Configuration: [`asana`](https://nango.dev/providers.yaml)
```
Comment (bastienbeurier) [inline docs-v2/integrations/all/gumroad.mdx]: Should be `gumroad`

File: docs-v2/mint.json [hunk 2]
```diff
@@ -177,6 +177,7 @@
                 "integrations/all/google-sheet",
                 "integrations/all/gorgias",
                 "integrations/all/greenhouse",
+		"integrations/all/gumroad",
```
Comment (bastienbeurier) [inline docs-v2/mint.json:180]: Looks like the indentation is off?

File: docs-v2/mint.json [hunk 3]
```diff
@@ -177,6 +177,7 @@
...
                 "integrations/all/gusto",
                 "integrations/all/hubspot",
```
Comment (bastienbeurier) [inline docs-v2/mint.json]: It looks like `hubspot` and `healthgorilla` are in the wrong order. Do you mind fixing this plz 🙏

File: packages/shared/providers.yaml [hunk 4]
```diff
@@ -300,10 +300,10 @@ greenhouse:
     auth_mode: OAUTH2
     authorization_url: https://api.greenhouse.io/oauth/authorize
     token_url: https://api.greenhouse.io/oauth/token
-health-gorilla:
+gumroad:
     auth_mode: OAUTH2
-    authorization_url: https://api.healthgorilla.com/oauth/authorize
-    token_url: https://api.healthgorilla.com/oauth/token
+    authorization_url: https://gumroad.com/oauth/authorize
+    token_url: https://api.gumroad.com/oauth/token
```
Comment (bastienbeurier) [inline packages/shared/providers.yaml:306]: Could you properly test that the OAuth flow works?

File: docs-v2/integrations/all/gumroad.mdx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+## Features
+
+| Feature                                                                          | Status                          |
+| -------------------------------------------------------------------------------- | ------------------------------- |
+| [Auth (OAuth)](/nango-auth)                                                      | ✅                              |
+| [Sync](/nango-sync)                                                              | ✅                              |
+| [Nango Proxy](/nango-sync/guides/nango-proxy)                          | ✅                              |
```
Comment (bastienbeurier) [inline docs-v2/integrations/all/gumroad.mdx]: It doesn't look like the proxy is supported, so the checkmark should be changed to "🚫 (time to contribute: &lt;1h)"

**Review by bastienbeurier — CHANGES_REQUESTED** [review]

**Comment by bastienbeurier** [comment]
Thank you for contributing this!

File: packages/shared/providers.yaml (hunk 4 above)
Comment (Tammilore) [reply packages/shared/providers.yaml:306]: Yup, this has been tested. It works.

File: docs-v2/integrations/all/gumroad.mdx (hunk 5 above)
Comment (Tammilore) [reply docs-v2/integrations/all/gumroad.mdx]: Thanks, will make all the changes.

**Review by bastienbeurier — APPROVED** [review]

**Pull request merged**